bindata/assets/kube-apiserver/pod.yaml

```diff
@@ -44,30 +39,18 @@ spec:
     args:
         - |
           LOCK=/var/log/kube-apiserver/.lock
-          echo -n "Acquiring exclusive lock ${LOCK}"
-          exec {LOCK_FD}>${LOCK} && flock -n "${LOCK_FD}" || {
+          echo -n "Acquiring exclusive lock ${LOCK} ..."
+          exec {LOCK_FD}>${LOCK} && flock --verbose -w 600 "${LOCK_FD}" || {
             echo "$(date -Iseconds -u) kubelet did not terminate old kube-apiserver before new one" >> /var/log/kube-apiserver/lock.log
             echo -n ": WARNING: kubelet did not terminate old kube-apiserver before new one."
-            # we didn't get an exclusive lock. We keep going with the risk to corrupt audit logs.
+            exit 1
```

Comment:
@sttts this is here after 10minutes pass and kubelet has still not terminated the process